docker-compose.yml

```diff
@@ -74,6 +76,7 @@ services:
       args:
         URL: "https://github.com/algorand/indexer"
         BRANCH: "master"
+        # Is there a reason why we're still pinning to a particular commit?
```

Comment:
Yes, this is pinned to a specific release. Based on the comment I guess this is the one that introduced Applications.